src/extension/features/general/colour-blind-mode/index.css

```diff
@@ -76,57 +17,227 @@ body {
+  --header_tbb_positive_background: var(--tk-colour-blind-positive) !important;
+  --label_positive: var(--tk-colour-blind-positive) !important;
+  --label_positive_active: var(--tk-colour-blind-positive-l090c090) !important;
+  --message_positive_accent: var(--tk-colour-blind-positive-l090c090) !important;
+  --message_positive_background: var(--tk-colour-blind-positive-l160c015) !important;
+  --message_positive_label: var(--tk-colour-blind-positive-l030c045) !important;
+  --status_positive: var(--tk-colour-blind-positive) !important;
 
-  --message_warning_background: var(--tk-colour-blind-warning-darkest) !important;
+  /* Warning */
+  --budget_balance_warning_background: var(--tk-colour-blind-warning) !important;
+  --budget_balance_warning_background_active: var(--tk-colour-blind-warning-l090c090) !important;
+  --budget_balance_warning_icon: #fff !important;
+  --budget_balance_warning_text: #fff !important;
+  --budget_bar_underfunded: var(--tk-colour-blind-warning) !important;
+  --budget_bar_underfunded_spending: var(--tk-colour-blind-warning-l125c065) !important;
+  --budget_bar_underfunded_spending_stripe: var(--tk-colour-blind-warning-l160c015) !important;
+  --label_warning: var(--tk-colour-blind-warning-l045c055) !important;
   --message_warning_accent: var(--tk-colour-blind-warning) !important;
-  --message_warning_label: var(--tk-colour-blind-warning) !important;
+  --message_warning_background: var(--tk-colour-blind-warning-l160c015) !important;
+  --message_warning_label: var(--tk-colour-blind-warning-l030c045) !important;
+  --status_cautious: var(--tk-colour-blind-warning) !important;
+}
 
-  --message_negative_background: var(--tk-colour-blind-negative-darkest) !important;
-  --message_negative_accent: var(--tk-colour-blind-negative) !important;
-  --message_negative_label: var(--tk-colour-blind-negative) !important;
+/* Square corners */
+body[tk-colour-blind-negative-square] .ynab-new-budget-available-number.negative {
+  border-radius: 0em !important;
+  -webkit-border-radius: 0em !important;
+}
+
+body[tk-colour-blind-positive-square] .ynab-new-budget-available-number.positive {
+  border-radius: 0em !important;
+  -webkit-border-radius: 0em !important;
+}
+
+body[tk-colour-blind-warning-square] .ynab-new-budget-available-number.cautious {
+  border-radius: 0em !important;
+  -webkit-border-radius: 0em !important;
 }
 
-/* Square edges for negative values */
-.ynab-new-budget-available-number.negative {
+body[tk-colour-blind-negative-square] .left-to-budget-is-negative .budget-header-totals-amount {
   border-radius: 0em !important;
   -webkit-border-radius: 0em !important;
 }
 
-.left-to-budget-is-negative .budget-header-totals-amount {
+body[tk-colour-blind-positive-square] .left-to-budget-is-positive .budget-header-totals-amount {
   border-radius: 0em !important;
   -webkit-border-radius: 0em !important;
 }
+
+/* Options menu */
+.tk-colour-blind-positive,
+.tk-colour-blind-warning,
+.tk-colour-blind-negative {
```

Comment:
I think we'll want a color picker icon or something in these guys which informs the user that they can change the color.